src/client/pages/Download/components/DownloadPageContent.tsx

```diff
@@ -6,6 +6,16 @@ import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
 import { useBreakpoint } from 'utils/hooks/useBreakpoint'
 import { TOP_BAR_HEIGHT } from 'components/TopBar'
 import { useFeature, Features } from 'utils/hooks/useFeature'
+import { Arrow } from 'components/icons/Arrow'
+import {
+  Description,
+  Header,
+  Image,
+  ImageFrame,
+  LinkCard,
+  Section,
+  Title,
+} from '../../Offer/ProductSelector/AdditionalProductCard'
```

Comment:
hmm I wonder if it's worth it to extract this to a global component so it's clear that it's being used in different pages

Reply:
Yes, perhaps! But then I think all cards should be global since they're used for the same feature, yes?

Reply:
Yeah sounds about right

Reply:
Fixed now 👍